module/input/GameController/src/GameControllerData.hpp

```diff
@@ -125,8 +127,10 @@ namespace module::input::gamecontroller {
     inline std::ostream& operator<<(std::ostream& os, const Robot& robot) {
         os << "\t\tPenalty state......: " << uint(robot.penaltyState) << std::endl
            << "\t\tPenalised time left: " << uint(robot.penalisedTimeLeft) << std::endl
+           << "\t\tNumber of warnings: " << uint(robot.numberOfWarnings) << std::endl
            << "\t\tYellow Card Count..: " << uint(robot.yellowCardCount) << std::endl
-           << "\t\tRed Card Count.....: " << uint(robot.redCardCount) << std::endl;
+           << "\t\tRed Card Count.....: " << uint(robot.redCardCount) << std::endl
+           << "\t\tGoalkeeper: " << uint(robot.goalKeeper) << std::endl;
```

Comment:
this should probs be bool(robot.goalKeeper) or something

Reply:
fixed the same way the other bool is done